src/global_logic.js

```diff
@@ -1975,7 +1970,7 @@ module.exports.treatSupportAbility = function (totals, chara) {
                     continue;
                 case "envoy_meditation":
                     var elements = module.exports.checkNumberOfElements(totals);
```

Comment:
Now (ind < 4) check was removed. the function may return 5 and 6? (for sub members)

I do not know yet about the "totals" data structure.
I worry is if a party like [dark, dark, dark, dark, water, fire] return 3
but front members elements are only dark.


```javascript
var elements = Math.max(4, module.exports.checkNumberOfElements(totals));
```

then code below does not need to check element >= 4 every times.
```javascript

totals[key]["normalBuff"] += (elements * 0.15);
totals[key]["DABuff"] += (elements * 0.10);
totals[key]["TABuff"] += (elements * 0.03);
```
for more clean up (hand write code here in comment form, not tested)